src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/FallbackConfigurationProvider.cs

```diff
@@ -59,7 +58,8 @@ public override bool TryResolveConfiguration(RazorConfigurationProviderContext c
                 return false;
             }
 
-            configuration = FallbackRazorConfiguration.SelectConfiguration(version);
+            var razorConfiguration = FallbackRazorConfiguration.SelectConfiguration(version);
+            configuration = new ProjectConfiguration(razorConfiguration, Array.Empty<OmniSharpHostDocument>());
```

Comment:
I don't understand what happened here.

Reply:
The output type is now `ProjectConfiguration` instead of `RazorConfiguration`. In the fall back case there's 0 chance for components so no point in trying to determine host documents from the Content item group.